projects/cps-ui-kit/src/lib/components/cps-scheduler/cps-scheduler.component.ts

```diff
@@ -203,14 +205,10 @@ export class CpsSchedulerComponent implements OnInit, OnChanges {
       if (!this.cron) this.cron = this._minutesDefault;
     }
 
-    this._handleModelChange(this.cron);
-
-    this.form = this._fb.group({
-      advanced: [
-        this.state.advanced.expression ?? '',
-        [this._validateAdvancedExpr]
-      ]
+    this.form.setValue({
+      advanced: this.state.advanced.expression ?? ''
     });
```

Comment:
Isn't this setValue call redundant here? Because the advanced value is set in `handleModelChange` on line 592.

Reply:
Fixed thanks for that :)